common/djangoapps/student/models.py

```diff
@@ -3118,6 +3137,95 @@ def should_celebrate_first_section(enrollment):
         except CourseEnrollmentCelebration.DoesNotExist:
             return False
 
+    @classmethod
+    def _get_today(cls):
+        """ Retrieve the value for today's date in the user's timezone """
+        # importing here to avoid a circular import
+        from lms.djangoapps.courseware.context_processor import user_timezone_locale_prefs
```

Comment:
should I try to refactor the user_timezone_locale_prefs function so the import here is not necessary? (though could end up having a lot of touchpoints)